### ISSUE TITLE
conflict with facebook DraweeView on function name

### ISSUE THREAD

**Issue by caojianfeng** [alexvasilkov/GestureViews#25]
I want create a class like this 
```
class ReaderCellView extends DraweeView implements GestureView{
//...
}
```

but both DraweeView and GestureView has function getController() 
and the return type is different.

And I think it is better to make the function of a Interface as special as we can.

**Comment by caojianfeng** [comment]
can we use getAlexGestureController as the name of the function ?

**Comment by alexvasilkov** [comment]
Yeah, it's sad that Fresco uses same method name. The only workaround I can suggest right now is to attach source code of this library and change method's name to whatever you like.

I will consider this issue in next major release, but it will not be very soon unfortunately.

**Comment by caojianfeng** [comment]
Thank you very much!

**Comment by alexvasilkov** [comment]
Thanks for your feedback, I will keep it open until I'll find a good way to fix it.

**Comment by Droppers** [comment]
Hi @alexvasilkov,

Is there any update on this?

**Comment by alexvasilkov** [comment]
I'm planing to implement this as part of future 3.x.x version since it requires breaking changes.
But I still can't share any specific date when it will be released.

Meanwhile I would suggest to use Glide or something similar if you want to use this library.
